.github/commitlint.config.mjs

```diff
@@ -0,0 +1,4 @@
+export default {
+    extends: ["@commitlint/config-conventional"],
+    ignores: [(message) => /^build\(deps\): bump .*$/m.test(message)],
```

Comment:
From now on, I will start all my PRs with `build(deps): bump <whatever>` :smiling_imp:.

On a bit more serious note, is it not possible to filter by author?

Reply:
Ahah let me add another rule :D 

I didn't find anyway to filter per author in commitlint directly